index.js

```diff
@@ -1,25 +1,25 @@
+'use strict';
+
 /**
  * Extract the @import/@require statements from a given stylus file's content
  *
  * @param  {String} fileContent
  * @return {String[]}
  */
 module.exports = function(fileContent) {
-  if (typeof fileContent === 'undefined') { throw new Error('content not given'); }
-  if (typeof fileContent !== 'string') { throw new Error('content is not a string'); }
-
-  var dependencies = [];
-  var importRegex = /\@(import|require)\s['"](.*)['"](\.styl)?/g;
+  if (typeof fileContent === 'undefined') throw new Error('content not given');
+  if (typeof fileContent !== 'string') throw new Error('content is not a string');
 
-  var matches;
+  const dependencies = [];
+  const importRegex = /@(?:import|require)\s['"](.*)['"](?:\.styl)?/g;
```

Comment:
Changed the groups to non-capturing while at it since we are not doing anything with them.